erdblick_app/app/parameters.service.ts

```diff
@@ -155,19 +210,19 @@ const erdblickParameters: Record<string, ParameterDescriptor> = {
         urlParam: true
     },
     osm: {
-        converter: val => val === 'true',
+        converter: val => val === 'true' || val === '1',
         validator: val => typeof val === 'boolean',
         default: true,
         urlParam: true
     },
     layers: {
-        converter: val => JSON.parse(val),
+        converter: val => decodeCompactBooleans(JSON.parse(val), 'layers'),
```

Comment:
Instead of the rather brittle `decodeCompactBooleans` function, I would propose to extend `validateObject` since it is informed about which fields are supposed to be booleans. Correspondingly, `validateObject` might be extended to also understand arrays like `["string", "number", "boolean", "boolean"]`, so it could be used here as well.